image_moo.php

```diff
@@ -201,13 +201,27 @@ public function check_gd()
 		}
 	}
 
+	//----------------------------------------------------------------------------------------------------------
+	// checks that we have an GD image loaded
+	//---
+
+	function _is_gd_image( $image ) {
```

Comment:
Quite an old library, but if it is going to support PHP 8 then this should be a private function now, without the _ prefix on the function name.